R/Accelerated_Failure_time_model.qmd

```diff
@@ -0,0 +1,133 @@
+---
+title: "Accelerated Failure Time Model"
+format: html
+editor: visual
+---
+
+The accelerated failure model is a survival analysis technique used to model the relationship between the event of interest (e.g., time to failure) and a set of predictor variables. It assumes that the covariates have a multiplicative effect on the hazard (rate of failure) function. In other words, the hazard rate is accelerated or decelerated by a factor that depends on the values of the covariates.
+
+## Mathematical Expression
+
+Mathematically, the AFT model can be expressed as:
+
+$\log(T) = X\beta + \sigma\varepsilon$
+
+Where:
+
+-   T is the survival time
+
+-   log(T) is the logarithm of the survival time
+
+-   X is a matrix of predictor variables
+
+-   β is a vector of coefficients representing the effects of the predictor variables on the logarithm of the survival time
+
+-   σ is a scaler quantity
+
+-   ε is the error term assumed to follow a specific distribution that depends on the assumed parametric form of the survival time.
+
+## Example of AFT model using "Log-Normal Distribution"
+
+```{r}
+library(survival)
+attach(lung)
+# Fit an AFT model using lognormal distribution
+model_aft <- survreg(Surv(time, status) ~ age + sex + ph.ecog, data = lung, dist = "lognormal")
+# Model summary
+summary(model_aft)
+```
+
+The summary output will provide the estimated coefficients, standard errors, and p-values for each predictor variable.
+
+## Acceleration Factor Calculation
+
+```{r}
+# Compute hazard ratios (exponentiated coefficients)
+hazard_ratios <- exp(coef(model_aft))
```

Comment:
In the context of the AFT model, use "acceleration factor" instead of "hazard ratio."The term "acceleration factor" is more appropriate as it describes how covariates affect the time to event by speeding up or slowing down the survival time, rather than affecting the hazard rate. Would suggest to update to this or something similar `# Compute acceleration factors (exponentiated coefficients)
acceleration_factors <- exp(coef(model_aft))
acceleration_factors`

Reply:
updated